spec/support/features/session_helper.rb

```diff
@@ -213,5 +213,143 @@ def session_store
       config = Rails.application.config
       config.session_store.new({}, config.session_options)
     end
+
+    def sign_up_user_from_sp_without_confirming_email(email)
```

Comment:
so is the plan to break this up in a future PR? I commented about doing that before and I remember that you replied but the thread is gone now...